manifests/bucketeer/charts/batch/templates/envoy-configmap.yaml

```diff
@@ -0,0 +1,273 @@
+              no_traffic_interval: 2s
+              timeout: 1s
+              unhealthy_threshold: 2
+          ignore_health_on_host_removal: true
+
+        - name: backend
+          type: strict_dns
+          lb_policy: round_robin
+          connect_timeout: 5s
+          dns_lookup_family: V4_ONLY
+          load_assignment:
+            cluster_name: backend
+            endpoints:
+              - lb_endpoints:
+                  - endpoint:
+                      address:
+                        socket_address:
+                          address: backend.{{ .Values.namespace }}.svc.cluster.local
+                          port_value: 9000
+          transport_socket:
+            name: envoy.transport_sockets.tls
+            typed_config:
+              '@type': type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext
+              common_tls_context:
+                alpn_protocols:
+                  - h2
+                tls_certificates:
+                  - certificate_chain:
+                      filename: /usr/local/certs/service/tls.crt
+                    private_key:
+                      filename: /usr/local/certs/service/tls.key
+          typed_extension_protocol_options:
+            envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+              '@type': type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
+              explicit_http_config:
+                http2_protocol_options: {}
+          ignore_health_on_host_removal: true
+
+      listeners:
+        - name: ingress
+          address:
+            socket_address:
+              address: 0.0.0.0
+              port_value: 9000
+          filter_chains:
+            - filters:
+                - name: envoy.filters.network.http_connection_manager
+                  typed_config:
+                    '@type': type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+                    access_log:
+                      name: envoy.access_loggers.file
+                      typed_config:
+                        '@type': type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+                        path: /dev/stdout
+                    codec_type: auto
+                    http_filters:
+                      - name: envoy.filters.http.health_check
+                        typed_config:
+                          '@type': type.googleapis.com/envoy.extensions.filters.http.health_check.v3.HealthCheck
+                          cluster_min_healthy_percentages:
+                            batch:
+                              value: 25
+                          headers:
+                            - name: :path
+                              string_match:
+                                exact: /health
+                          pass_through_mode: false
+                      - name: envoy.filters.http.router
```

Comment:
Please add the `typed_config`.

https://github.com/bucketeer-io/bucketeer/pull/459/files#diff-6c6dfb590e04fdd74ea9cf9c5e456da2a4cea085da6a192c92dc0e67cf6ed317R181-R182